### ISSUE TITLE
New Dataset Page: Report an issue with this dataset button

### ISSUE THREAD

**Issue by t83714** [magda-io/magda#503]
This issue about implementing the `Report an issue with this dataset` button for the new dataset page:

![image](https://user-images.githubusercontent.com/674387/36648711-0f4b4854-1aeb-11e8-8b7d-86184ef464f5.png)

The button is part of `New Dataset Page Design` (Tracked as issue #500).

Created a new issue in order to track this functionality separately.

**More requirement information is required before starting to work on this item.**

**Comment by AlexGilleran** [comment]
Replaced by "Ask a question about this dataset"
